src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetPackageFileService.cs

```diff
@@ -81,7 +80,6 @@ public NuGetPackageFileService(
             AuthorizationServiceClient authorizationServiceClient,
             INuGetTelemetryProvider nuGetTelemetryProvider)
         {
-            _options = options;
```

Comment:
I think the parameter `ServiceActivationOptions options` can also be removed from the constructor and its callers. It can be done in a future PR.